tests/unit/OptionsListUtilsTest.js

```diff
@@ -199,17 +199,17 @@ describe('OptionsListUtils', () => {
         // When we call getNewChatOptions() with no search value
         let results = OptionsListUtils.getNewChatOptions(REPORTS, PERSONAL_DETAILS, '');
 
-        // Then no reports should be returned, only personalDetails and all the personalDetails should be returned
-        // minus the currently logged in user
-        expect(results.recentReports.length).toBe(0);
-        expect(results.personalDetails.length).toBe(_.size(PERSONAL_DETAILS) - 1);
+        // We should expect maxmimum of 5 recent reports to be returned
+        // minus the currently logged in user and recent reports count
+        expect(results.recentReports.length).toBe(5);
+        expect(results.personalDetails.length).toBe(_.size(PERSONAL_DETAILS) - 1 - 5);
 
         // Then the result which has an existing report should also have the reportID attached
         const personalDetailWithExistingReport = _.find(
             results.personalDetails,
-            personalDetail => personalDetail.login === 'reedrichards@expensify.com',
+            personalDetail => personalDetail.login === 'peterparker@expensify.com',
```

Comment:
Does this change the test results? Not sure why `expect(personalDetailWithExistingReport.reportID).toBe(2);` now expects 2 instead of 3

Reply:
Yes it changes. `reedrichards@expensify.com` will be returned in `recentReports` not in `personalDetails`.

Either way, it checks if the array contains reportID